Accessibility/VirtualContent/explainer.md

```diff
@@ -129,6 +129,72 @@ window.addEventListener("scroll", on_scroll_changed);
 13. Eventually, the backing server sends a content payload which also contains a signal that the end of the document has been reached. Script code removes the `aria-virtualcontent` attribute from the `main` element to indicate there is no more virtualized content.
 14. The next time the user attempts to navigate to the next heading, the AT searches for a virtual content container, finds none, and announces there are no further headings in the document.
 
+## Example Use 2: Tabular data
+
+This example illustrates one use of `aria-virtualcontent` for virtualized content in the inline direction. The document in this scenario is a table of bug reports with the following columns: report ID, report date, status, assigned to, title, fix date, fixed by. The last two columns have been virtualized out of the DOM based on the viewport width. When realized, these two columns do not have any data for bugs that are not yet fixed.
```

Comment:
Can we make this example way simpler? I'm getting lost in the weeds of all the steps involved. It would be great if this example could work with all virtual content indicators on the table itself, e.g. `<table aria-virtualcontent="block-end inline-end"`, and we could show how the AT could use this signal to move in one particular direction. For example, maybe the user asked for the next cell after the last realized column. Or the AT wants to proactively surface that there are columns which aren't loaded yet.

Reply:
What I was attempting to show is that content can be tuned such that the AT doesn't have to scoll back and forth searching for cells that don't exist, but I agree that does make the example really complex. I'll see what I can do to simplify it.

Reply:
Done